weatherman.py

```diff
@@ -0,0 +1,192 @@
+import argparse
+import calendar
+from colorama import Fore, Style
+import csv
+import os
+
+def convert_month(month_input):
+    try:
+        month_num = int(month_input)
+        if 1 <= month_num <= 12:
+            return calendar.month_abbr[month_num]
+        else:
+            raise ValueError("Invalid month number")
+    except ValueError:
```

Comment:
What is the purpose of having both the exception block and the if block, especially when they are essentially performing the same action?

Please find a better way to handle this